docs/source/author/understanding_configuration_files.rst

```diff
@@ -0,0 +1,63 @@
+Understanding configuration files
+##############################################################################
+
+There is a lot of content within the CS Unplugged project.
+We split this content across many files and configuration files are the things
+that bring everything together.
+Understanding how to read and write these configuration files is important
+for working on this project, because these files are used for configuring the
+content data when stored in the system database.
+
+This page aims to give a brief tutorial on YAML files, so you can modify
+configuration files within this project.
+
+Here is an example YAML file:
+
+.. code-block:: yaml
+
+  learning-outcomes: learning-outcomes.yaml
+  programming-exercises-structure: programming-exercises-structure.yaml
+
+  topic_structure_files:
+    - binary-numbers/structure.yaml
+    - error-detection-and-correction/structure.yaml
+
+YAML files are mostly made up of key/value pairs, often called a dictionary
+within programming languages.
+This configuration file contains the follow three key/value pairs:
+
+- The key ``learning-outcomes`` points to the text string value
+  ``learning-outcomes.yaml``
+- The key ``programming-exercises-structure`` points to the text string value
+  ``programming-exercises-structure.yaml``
+- The key ``topic_structure_files`` points to the ordered list of text strings
+  values ``binary-numbers/structure.yaml`` and
+  ``error-detection-and-correction/structure.yaml``
```

Comment:
is there a reason why you have emphasised that these are "text strings"? Why not just "points to the file <filename>"
We as programmers know that it's a string, but authors just need to know that it matches a file name.